org.opent2t.sample.thermostat.superpopular/com.wink.thermostat/js/thingTranslator.js

```diff
@@ -214,6 +218,110 @@ class Translator {
                 return response.targetTemperatureLow.temperature;
             });
     }
+
+    subscribe(options) {
```

Comment:
>subscribe [](start = 4, length = 9)

It would be helpful to document the flow somehow so folks understand the pubsubhubbub model and call flow